src/text_engine/boundaries/Room.java

```diff
@@ -136,22 +133,6 @@ public boolean addItem(Item item) {
     }
 
     /**
-     * Examine the room, collecting all available objects into a string.
-     *
-     * @return A concatenation of all items' descriptions and door locations
-     */
-    public String examine() {
-        String result = "";
-        for (Item item : this.items) {
-            result += item.toString() + "\n";
-        }
-        for (Door door : this.doors) {
-            result += door.toString() + "\n";
-        }
-        return result;
-    }
-
-    /**
```

Comment:
This is being taken care of in `Printers` and `Interactive` whatsits, so there's no need for this any more.